sub-packages/bionemo-geneformer/tests/bionemo/geneformer/scripts/test_pydantic_train.py

```diff
@@ -117,9 +126,9 @@ def test_pretrain_cli(tmpdir):
     assert (result_dir / "test-experiment").exists(), "Could not find test experiment directory."
 
 
-def test_finetune_cli(tmpdir):
+def test_finetune_cli(tmpdir, data_path):
     """Uses CLI to invoke the entrypoint"""
-    data_path: Path = load("single_cell/testdata-20240506") / "cellxgene_2023-12-15_small" / "processed_data"
+    # data_path: Path = load("single_cell/testdata-20240506") / "cellxgene_2023-12-15_small" / "processed_data"
```

Comment:
remove comment?

Reply:
nit picking github etiquette -- please just write "removed" or "fixed" and allow Steven to resolve, unless you both met and agreed that you (Savitha) can resolve it.